src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,405 @@
+| Resolution   | REQUIRED if `res` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the resolution keyword.         |
+| Density      | REQUIRED if `den` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the density keyword.            |
+
+Examples:
+
+```Text
+func_loc/
+    sub-001/
+        func/
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.nii.gz
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.json
+```
+
+```Text
+manual_masks/
+    sub-001/
+        anat/
+            sub-001_desc-tumor_mask.nii.gz
+            sub-001_desc-tumor_mask.json
+```
+
+## Segmentations
+
+A *segmentation* is a labeling of a spatial image with anatomical structures
+such that each location (for example, a voxel or a surface vertex) is identified
+with a label or a combination of labels.
+A *discrete segmentation* represents each structure with a unique integer
+label.
+A *probabilistic segmentation* represents each structure as values between
+0 and 1 (inclusive) at each location in the image, and one volume/frame per
+structure may be concatenated in a single file.
+
+Segmentations may be defined in a volume (labeled voxels), a surface (labeled
+vertices) or a combined volume/surface space.
+
+The following section describes discrete and probabilistic segmentations of
+volumes, followed by discrete segmentations of surface/combined spaces.
+Probabilistic segmentations of surfaces are currently [unspecified][].
+
+The following metadata fields apply to all segmentation files:
+
+| **Key name** | **Description**                                                                                                                        |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Manual       | OPTIONAL. Boolean. Indicates if the segmenation was performed manually or via an automated process                                     |
+| Atlas        | OPTIONAL. Which atlas (if any) was used to derive the segmentation.                                                                    |
+| Resolution   | REQUIRED if `res` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the resolution keyword. |
+| Density      | REQUIRED if `den` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the density keyword.    |
+
+### Discrete Segmentations
+
+Discrete segmentations of brain tissue represent multiple anatomical structures
+(such as tissue class or Brodmann area) with a unique integer label in a 3D volume.
+See [Common image-derived labels](#common-image-derived-labels) for interpretation
+how integer values map to anatomical structures.
```

Comment:
Thanks for noticing this weird wording, @sanjulamadurapperuma. I think I would prefer:

```suggestion
See [Common image-derived labels](#common-image-derived-labels) for a description
of how integer values map to anatomical structures.
```

WDYT?

Reply:
Yes, I agree. This suggestion is better